cpp/ps0001/ps0001_soln.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+  // prompting user for 3 numbers to add 1.5/2
+  unsigned short num1, num2, num3;
+  cout << "please enter first number \n";
+  cin >> num1;
+  cout << "please enter second number \n";
+  cin >> num2;
+  cout << "please enter thired number \n";
+  cin >> num3;
```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add